### PR TITLE
NOISSUE- Add country filter to telemetry filters

### PR DESCRIPTION
**Pull request by 1998-felix**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Adds country filters to retrieve methods 

### Which issue(s) does this PR fix/relate to?
Put here Resolves ultravioletrs/issues#268 to auto-close the issue that your PR fixes (if such)

### Notes

### CODE REVIEW

**Review by SammyOina — CHANGES_REQUESTED** [review]
update openapi spec

File: timescale/timescale.go [hunk 1]
```diff
@@ -160,6 +160,9 @@ func generateQuery(filters callhome.TelemetryFilters) (string, map[string]interf
 	case !filters.To.IsZero():
 		queries = append(queries, "time <= :to")
 		params["to"] = filters.To
+	case filters.Country != "":
```
Comment (SammyOina) [inline timescale/timescale.go:163]: remove from switch case so that filters can be applied in combination

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Comment by SammyOina** [comment]
> Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to. Please review the [CONTRIBUTING.md](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md) file for detailed contributing guidelines.
> 
> ### What does this do?
> Adds country filters to retrieve methods
> 
> ### Which issue(s) does this PR fix/relate to?
> Put here `Resolves ultravioletrs/issues#268` to auto-close the issue that your PR fixes (if such)
> 
> ### List any changes that modify/break current functionality
> ### Have you included tests for your changes?
> ### Did you document any new/modified functionality?
> ### Notes

remove quotations around resolves ....

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**